Core/Utilities/TimeLeft/MJFTimeLeft.py

```diff
@@ -73,16 +73,14 @@ def getResourceUsage( self ):
       cpuLimit = int( urllib.urlopen(jobFeaturesPath + '/cpu_limit_secs').read() )
     except:
       self.log.warn( 'Could not determine cpu limit from $JOBFEATURES/cpu_limit_secs' )
-      cpuLimit = wallClockLimit
 
     wallClock = int(time.time()) - jobStartSecs
-    # We cannot get CPU usage from MJF, so for now use wallClock figure
-    cpu = wallClock
+    # We cannot get CPU usage from MJF
       
     consumed = {'CPU':cpu, 'CPULimit':cpuLimit, 'WallClock':wallClock, 'WallClockLimit':wallClockLimit}
     self.log.debug( consumed )
 
-    if cpu and cpuLimit and wallClock and wallClockLimit:
+    if cpuLimit and wallClock and wallClockLimit:
```

Comment:
I want to keep the same behaviour as before. I removed the constraint on cpu because it's normal to have it missing (from the comment in the source). I left the other constraints that correspond to MJF errors because I can't test with MJF very carefully.

I realize we could remove the constraint on wallClock because it is unconditionnally computed at line 77. I'll add a commit if you agree.